src/platform/Linux/ConnectivityManagerImpl.h

```diff
@@ -126,6 +126,17 @@ class ConnectivityManagerImpl final : public ConnectivityManager,
     bool _IsWiFiStationProvisioned();
     void _ClearWiFiStationProvision();
     bool _CanStartWiFiScan();
+
+    WiFiAPMode _GetWiFiAPMode(void);
```

Comment:
please remove the void in `(void)` - we had a clang tidy pass that already removed it everywhere else.

Reply:
Acked